src/components/HelpPage/HelpPage.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { HelpPageDisplayOptions } from '@cortexapps/backstage-plugin-extensions';
+import { isUndefined } from 'lodash';
+import { HelpLinksContent } from './HelpLinksContent';
+
+export interface HelpPageProps {
+  helpPage: HelpPageDisplayOptions;
+}
+
+export const HelpPage = ({ helpPage }: HelpPageProps) => {
+  return (
+    <>
+      {!isUndefined(helpPage.links) && (
+        <HelpLinksContent links={helpPage.links} />
+      )}
+    </>
+  );
```

Comment:
We should get rid of the fragment here and simplify this
```suggestion
  return isUndefined(helpPage.links) ? null : (
    <HelpLinksContent links={helpPage.links} />
  );
```

Reply:
apologies, will come back!